.github/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+## Title (delete after following this rule)
+
+Your PR title should be prefixed with the Jira ticket ID, e.g. "ADX-123 very important feature".
+
+## Description
+
+Please include a summary of the change(s).
+You should also include motivation: why was this change required and what problem it solves.
```

Comment:
> You should also include motivation: why was this change required and what problem it solves.
Maybe this should be part of the Jira ticket? In my head, Jira's still the primary source of information about work.
How exactly to define info stored in the GitHub PR? Maybe none of the issue descriptions should be here?

Some people add some screenshots for UI work they do in the PR and this is really helpful during the review. Especially for small changes.